src/src/Helpers/Hardware.cpp

```diff
@@ -1607,7 +1607,7 @@ void initAnalogWrite()
   #if defined(ESP32)
   for(uint8_t x = 0; x < 16; x++) {
     ledChannelPin[x] = -1;
-    ledChannelFreq[x] = ledcSetup(x, 0, 10); // Clear the channel
+    ledChannelFreq[x] = ledcSetup(x, 1000, 10); // Clear the channel
```

Comment:
So this is the actual fix?

Reply:
No, only one part of it, see other commits. Only when I change all three in Hardware.cpp and the one in Network.cpp it works.

Reply:
I just don't see why `pinMode(pin, OUTPUT);` breaks it. As well for GPIO -> LED in HW (Network.cpp) as well as with just the pwm command (Hardware.cpp).